lib/util/SparqlExpressionEvaluator.js

```diff
@@ -60,8 +60,10 @@ evaluators = {
         operator = operators[operatorName];
     if (!operator)
       throw new UnsupportedOperatorError(operatorName);
-    if (operator.length !== expression.args.length)
-      throw new InvalidArgumentsNumberError(operatorName, expression.args.length, operator.length);
+    var minArgs = operator.length - (operator.optionalArguments || 0);
```

Comment:
Is the subtraction really necessary? Maybe we should just adjust the definition of `minArgs` for operators that have optional arguments.